contrib/datastore/block/driver/factory.go

```diff
@@ -16,25 +16,22 @@ package driver
 
 import (
 	backendpb "github.com/opensds/multi-cloud/backend/proto"
-	excep "github.com/opensds/multi-cloud/block/pkg/exception"
-	log "github.com/sirupsen/logrus"
+	exp "github.com/opensds/multi-cloud/s3/pkg/exception"
```

Comment:
Why specific s3? Why cannot we move it to common

Reply:
Yes, We need to refactor the duplicate code for maintaing Exceptions across services. As if now I have updated to your's block code Exception.